client/src/lang/ar_SA/errors.json

```diff
@@ -29,7 +29,7 @@
   "ErrNoNodes": "No nodes available",
   "ErrNoTemplates": "No templates available",
   "ErrPasswordRequirements": "Password must be at least 8 characters",
-  "ErrUsernameRequirements": "Username must be at least 3 characters and only contain alphanumerics, _, or -",
+  "ErrUsernameRequirements": "Username must be at least 5 characters and only contain alphanumerics, _, or -",
```

Comment:
This must be done via the translation site, not manually.

Reply:
that actually shouldn't be there, that is part of another fix, I think I messed up that branch

Reply:
should be only one commit now with the fix to the valid username issue